src/fontra/client/core/glyph-controller.js

```diff
@@ -651,6 +655,33 @@ export class StaticGlyphController {
       }
     }
 
+    if (backgroundImageIndices.length) {
+      const backgroundImage = this.instance.backgroundImage;
+      if (!backgroundImage) {
+        return unionRect(...selectionRects);
+      }
+      if (!fontController) {
+        return unionRect(...selectionRects);
+      }
+      const image = fontController.getBackgroundImageCached(backgroundImage.identifier);
+      if (!image) {
+        return unionRect(...selectionRects);
+      }
+
+      const affine = decomposedToTransform(backgroundImage.transformation)
+        .translate(0, image.height)
+        .scale(1, -1);
+
+      const pt1 = affine.transformPointObject({ x: 0, y: 0 });
```

Comment:
In rectangle.js, please add a function `rectToPoints()` that returns an array of the four corner points. From the image width and height you can most easily construct a rect with the `rectFromArray([xMin, yMin, xMax, yMax])` function.

Reply:
done. I also added unittests for rectToPoints.